docs/aap-api-token.md

```
@@ -0,0 +1,241 @@
+```
+
+## Retrieving AAP URL from Cluster
+
+The AAP URL can be found via the route in the awx namespace:
+
+```bash
+kubectl get routes -n awx
+```
+
+Example output:
+```
```

markdownlint-cli2 (0.22.0)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)